### PR TITLE
Add octomap to cmakelist

### PR DESCRIPTION
**Pull request by mpowelson**

### CODE REVIEW

File: tesseract/tesseract_geometry/CMakeLists.txt [hunk 1]
```diff
@@ -5,6 +5,9 @@ find_package(Eigen3 REQUIRED)
 find_package(console_bridge REQUIRED)
 find_package(tesseract_common REQUIRED)
 
+find_package(octomap REQUIRED)
+include_directories(${OCTOMAP_INCLUDE_DIRS})
```
Comment (Levi-Armstrong) [inline tesseract/tesseract_geometry/CMakeLists.txt]: Please move this to the target that requires it.

**Review by Levi-Armstrong — CHANGES_REQUESTED** [review]
Also need to add dependency to the cmake file in the cmake folder.

File: tesseract/tesseract_geometry/CMakeLists.txt (hunk 1 above)
Comment (mpowelson) [reply tesseract/tesseract_geometry/CMakeLists.txt]: Done

**Comment by mpowelson** [comment]
@Levi-Armstrong Changes made

**Review by Levi-Armstrong — APPROVED** [review]

**Pull request merged**